tests/admin-mode_test.go

```diff
@@ -0,0 +1,194 @@
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	//Checking the status of operator which is already installed and running
+	operator, err := client.AppsV1().Deployments(chaosTypes.ChaosNamespace).Get("litmus", metav1.GetOptions{})
+	Expect(err).To(BeNil(), "Failed to get operator")
+	count := 0
+	for operator.Status.UnavailableReplicas != 0 {
+		if count < 50 {
+			fmt.Printf("Unavaliable Count: %v \n", operator.Status.UnavailableReplicas)
+			operator, err = client.AppsV1().Deployments(chaosTypes.ChaosNamespace).Get("litmus", metav1.GetOptions{})
+			Expect(err).To(BeNil(), "Failed to get operator")
+			time.Sleep(5 * time.Second)
+			count++
+		} else {
+			Fail("Operator is not in Ready state Time Out")
+		}
+	}
+	fmt.Println("Chaos Operator is up and Running")
+})
+
+//BDD Tests for testing admin mode functionality
+//Checking for the creation of runner pod for application in same namespace
+var _ = Describe("BDD of operator reconcile resiliency check", func() {
+
+	// BDD TEST CASE 1
+	Context("Check for litmus components", func() {
+
+		It("Should check for creation of runner pod", func() {
+
+			//Create Namespace for the test
+			By("Creating namespace for the test")
+			cmd := exec.Command("kubectl", "create", "ns", "test")
+			cmd.Stderr = os.Stderr
+			err := cmd.Run()
+			if err != nil {
+				log.Fatalf("cmd.Run() failed with %s\n", err)
+			}
+
+			//Creating application for pod-delete in test namespace
+			By("Creating deployment for pod-delete chaos")
+			err = exec.Command("kubectl", "run", "adminapp", "--image=nginx").Run()
+			Expect(err).To(BeNil(), "Failed to create deployment")
+			fmt.Println("Test Application is created")
+
+			//Installing RBAC for experiment that is pod-delete
+			rbacPath := "https://raw.githubusercontent.com/litmuschaos/pages/master/docs/litmus-admin-rbac.yaml"
+			err = exec.Command("wget", "-O", experimentName+"-sa.yaml", rbacPath).Run()
+			Expect(err).To(BeNil(), "Fail to fetch rbac")
+			err = exec.Command("sed", "-i", `s/namespace: litmus/namespace: test/g`, experimentName+"-sa.yaml").Run()
+			Expect(err).To(BeNil(), "Fail to modify rbac")
+			err = exec.Command("kubectl", "apply", "-f", experimentName+"-sa.yaml").Run()
+			Expect(err).To(BeNil(), "Fail to create RBAC")
+			fmt.Println("Rbac has been created successfully !!!")
+
+			//Creating Chaos-Experiment for pod delete
+			By("Creating Experiment for pod delete")
+			err = exec.Command("wget", "-O", "pod-delete.yaml", "https://hub.litmuschaos.io/api/chaos?file=charts/generic/pod-delete/experiment.yaml").Run()
+			Expect(err).To(BeNil(), "fail to get chaos experiment")
+			err = exec.Command("sed", "-i", `s/litmuschaos\/ansible-runner:latest/`+chaosTypes.ExperimentRepoName+`\/`+chaosTypes.ExperimentImage+`:`+chaosTypes.ExperimentImageTag+`/g`, "pod-delete.yaml").Run()
+			Expect(err).To(BeNil(), "Failed to edit chaos experiment yaml")
+			err = exec.Command("kubectl", "apply", "-f", "pod-delete.yaml", "-n", "test").Run()
```

Comment:
namespace should be chaosTypes.ChaosNamespace

Reply:
`chaosTypes.ChaosNamespace` is litmus. I need this in `test` ns